collectors/python.d.plugin/python.d.conf

```diff
@@ -29,6 +29,7 @@ gc_interval: 300
 
 # apache_cache has been replaced by web_log
 # adaptec_raid: yes
+alarms: no
```

Comment:
```suggestion
# alarms: no
```

this doesn't allow us to swith from disabled to enabled or vice versa (add/remove `disabled_by_default`).